charts/internal/cri-resmgr-installation/templates/installaton-configmap.yaml

```diff
@@ -59,6 +59,11 @@ data:
         sed -i 's!fallback.cfg --force-config /etc/cri-resmgr/force.cfg"!fallback.cfg"!' /var/host/etc/default/cri-resource-manager
       fi
 
+      # Warning if previous state exists
+      if [[ -d /var/host/var/lib/cri-resmgr ]]; then
+        echo 'WARNING! previous state of cri-resmgr still exists in hosts /var/lib/cri-resmgr!'
```

Comment:
I added something like this:
```
        echo 'WARNING! previous state of cri-resmgr still exists in hosts /var/lib/cri-resmgr!'
        echo '         cri-resource-manager was not properly uninstalled, cached config will be used instead of fallback.cfg'
        echo '         To fix just restart this pod or manually remove /var/lib/cri-resmgr on host.'
```